Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -55,8 +55,8 @@ class PrPull < AbstractCommand
         flag   "--message=",
                depends_on:  "--autosquash",
                description: "Message to include when autosquashing revision bumps, deletions and rebuilds."
-        flag   "--artifact=",
-               description: "Download artifacts with the specified name (default: `bottles`)."
+        flag   "--artifact-pattern=", "--artifact=",
+               description: "Download artifacts with the specified pattern (default: `bottles{,-*}`)."
```

Comment:
To preserve backward compatibility:

- The `--artifact=` flag is kept as an alias of `--artifact-pattern`.
- The default pattern still allows the plain `bottles` artifact name.